src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/Validation/ValidationVisitor.cs

```diff
@@ -153,6 +153,17 @@ protected virtual bool ValidateNode()
             }
         }
 
+        /// <summary>
+        /// Gets an indication whether the <paramref name="entry"/> should be validated.
+        /// </summary>
+        /// <param name="entry"><see cref="ValidationEntry"/> to check.</param>
+        /// <param name="parentEntry"><see cref="ValidationEntry"/> containing <paramref name="entry"/>.</param>
+        /// <returns><c>true</c> if <paramref name="entry"/> should be validated; <c>false</c> otherwise.</returns>
+        protected virtual bool ShouldValidateEntry(ValidationEntry entry, ValidationEntry parentEntry)
+        {
+            return true;
+        }
```

Comment:
Outdated comment. No longer have (or need) this `ValidationVisitor` extensibility point. But, I'll assume you love the `IShouldValidate` interface 😺